public/app/features/profile/ReactProfileWrapper.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { UserProvider } from 'app/core/utils/UserProvider';
+import { UserProfileEditForm } from './UserProfileEditForm';
+import { SharedPreferences } from 'app/core/components/SharedPreferences/SharedPreferences';
+import { config } from '@grafana/runtime';
+
+export const ReactProfileWrapper = () => (
```

Comment:
Can you please rename this to `UserProfileEditor`? And the angular wrapper as well

Reply:
But this won't just wrap that component- it will wrap all of the components I convert in this page as I go. Still change it?